recipe/meta.yaml

```diff
@@ -8,70 +8,78 @@ package:
 source:
   - folder: dist
     url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: 4a41c5b3a65ed92e469d51b6fba3779301850ea2e352afcf9e36c46f21ee14a9
-    patches:
-      # we skip a single test because of our kernel version
-      - 0002-skip-test-if-reuseport-is-not-available-dist.patch
+    sha256: a0aeffe2fb1fdf374a8e4b471444f0f3ac4fb9f5a5b542b48824475e0042a5a6
+#    patches:
```

Comment:
Should we remove the patch code instead of commenting it out? What value do we get retaining this? And if there is some value, should we be adding it to the comment?

Reply:
Yes, they are redundant. I removed them.